$NetBSD: patch-ac,v 1.3 2013/02/04 20:31:44 markd Exp $

Use $host_alias in the gcc-libdir path so it matches ${MACHINE_GNU_PLATFORM}

--- configure.orig	2010-04-22 04:07:13.000000000 +0000
+++ configure
@@ -4647,7 +4647,7 @@ _ACEOF
    { $as_echo "$as_me:${as_lineno-$LINENO}: result: $gcc_version" >&5
 $as_echo "$gcc_version" >&6; }
 
-   DEST_LIBDIR=$prefix/lib/gcc-lib/$host/$gcc_version
+   DEST_LIBDIR=$prefix/lib/gcc-lib/$host_alias/$gcc_version
    SAVE_DIR=`pwd`
    PREFIX=$prefix
 
